Extensions/xkit_patches.js

```diff
@@ -1494,12 +1494,13 @@ XKit.tools.getParameterByName = function(name){
 			post: function(obj) {
 
 				var m_return = {};
+				var post_data = JSON.parse($(obj).attr('data-json'));
```

Comment:
I'd prefer this to have a fallback like line 1073 above where it's JSON.parse(asdf || "{}"). This would make the following if-statement detect the error where 'data-json' is undefined instead of having the parse throw an uncaught exception.